core-build/gulp-core-build/src/index.ts

```diff
@@ -230,14 +230,14 @@ export function watch(watchMatch: string | string[], taskExecutable: IExecutable
                 }
                 return _finalizeWatch();
               })
-              .catch((error) => {
+              .catch((error: Error | string) => {
```

Comment:
>| string [](start = 35, length = 8)

Can this one ever be string type?

Reply:
+1

Our coding standard is to always throw subclasses of Error.  If someone throws a plain string, that is a bug that should be fixed immediately.

If an OSS library is throwing these bad objects, we should have special code to wrap it, with a comment explaining why it's needed.

---
In reply to: [147839652](https://github.com/Microsoft/web-build-tools/pull/419#discussion_r147839652) [](ancestors = 147839652)

Reply:
I have proactively approved this PR, but please fix this issue (and any other occurrences of it) before merging

---
In reply to: [147844044](https://github.com/Microsoft/web-build-tools/pull/419#discussion_r147844044) [](ancestors = 147844044,147839652)

Reply:
I double checked and it looks like it cannot be a string type. It is normalized in the base GulpTask, and serial and parallel properly return Error objects